### PR TITLE
DDP-5253 removing stay informed section and adding rally link

### PR DESCRIPTION
**Pull request by zyme**
Hotifx for applying DDP-5253 to current prod

### CODE REVIEW

File: ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html [hunk 1]
```diff
@@ -24,12 +24,6 @@
                         Navigation.WhoWeAre
                     </a>
                 </li>
-                <li class="navigation__item">
-                    <a class="header-link Link_no-wrap" routerLink="/" fragment="stay-informed"
-                        (click)="scrollToAnchor('stay-informed')" translate>
-                        Navigation.StayInformed
-                    </a>
-                </li>
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html:32]: You've forgotten to remove one more mailing list: 79-84 lines.
And you have to remove the mailing list from the footer.

Comment (zyme) [reply ddp-workspace/projects/ddp-testboston/src/app/components/header/header.component.html:32]: Oooh whoops!  Thank you.

**Review by MocanaAtBroad — APPROVED** [review]
Changes look good.
Couple of quick observations:
1) Language for NoCode is still missing for the other languages (e.g, French, Arabic. etc). This might be OK.
2) In mobile display, the "I have an invitation code" button and the accompanying "I don't have an invitation" link are left justified. In the non-updated languages these are center justified. They both look good to me.

**Comment by zyme** [comment]
Thanks @MocanaAtBroad!  Indeed only a few languages are translated for now (as intended).

**Pull request merged**